File: src/test/testdata/inspections/unsafeIterators/IntSet.java
package main;

import com.badlogic.gdx.utils.IntSet;

class IntSetTest {

  void test() {
    new IntSet().iterator();

    IntSet set = new IntSet();

    IntSet.IntSetIterator i =  <warning>set.iterator()</warning>;

    for (int x: <warning>set.iterator()</warning>.toArray().toArray()) {

    }


  }

}

File: src/test/testdata/inspections/unsafeIterators/IntSet.kt
package main

import com.badlogic.gdx.utils.IntSet

@Suppress("UNUSED_VARIABLE", "UNCHECKED_CAST", "USELESS_CAST")
class IntSetTest {

  fun test() {
    <warning>IntSet().iterator()</warning>

    val set = IntSet()

    val i = <warning>set.iterator()</warning>

    for (x in <warning>set.iterator()</warning>.toArray().toArray()) {

    }


  }

}

